jb/itch5/array_based_order_book.cpp

```diff
@@ -1,20 +1,65 @@
-#include <jb/itch5/price_field.hpp>
-#include <jb/itch5/quote_defaults.hpp>
-#include <jb/feed_error.hpp>
+#include <jb/itch5/array_based_order_book.hpp>
```

Comment:
Sorry, I missed this in the last review.  Use "" for this #include... That way clang-format knows that it should be the first one.